addons/mail/static/src/components/media_preview/media_preview.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.MediaPreview" owl="1">
+        <div class="o_MediaPreview position-relative d-flex justify-content-center">
+            <video class="o_MediaPreview_videoDisplay shadow rounded bg-dark" height="480" width="640" autoplay="" t-ref="video"/>
```

Comment:
Width and height can't be in css instead? This doesn't look very responsive at all.